sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/models/ier12-integration.model.ts

```diff
@@ -126,9 +126,9 @@ export enum ScholasticStandingCode {
    */
   EC = "EC",
   /**
-   * Change in intensity.
+   * School transfer.
    */
-  CI = "CI",
+  ST = "ST",
```

Comment:
Please update the IER spreadsheet if not already updated.

Reply:
The spreadsheet and word doc were updated as per the PR description 😄